datafusion/core/src/physical_plan/file_format/parquet/page_filter.rs

```diff
@@ -222,6 +222,11 @@ impl PagePruningPredicate {
         file_metrics.page_index_rows_filtered.add(total_skip);
         Ok(Some(final_selection))
     }
+
+    /// Returns the number of filters in the [`PagePruningPredicate`]
+    pub fn filter_number(&self) -> usize {
```

Comment:
Given the use of this code, I think using `is_empty()` rather than `len()` would be clearer

Reply:
I think need the number of filters in future 😆